### PR TITLE
vscode tizen extension user docs update for Rpk app

### PR DESCRIPTION
**Pull request by rakeshkumarrajus**
Signed-off-by: Rakesh Kumar S/Advance–Software Development Tool /SRI-Bangalore/Staff Engineer/Samsung Electronics <rakeshraju.s@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by sangwook96 — APPROVED** [review]
Great, Thank you very much!!

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+   ![Run application](media/rpk_run_app1.png)
+
+   ![application](media/rpk_run_app2.png)
+
+   > [!NOTE] 
+   > Resource project is Installed, But Resource project cannot be launched by Run info popup will be displayed.
+
+
+# Tizen workspace yaml Validation
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:82]: I think second heading is more appropriate here, as an md file can only have one First heading or Title on Tizen Docs.
```suggestion
## Tizen workspace YAML validation
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+1. Create Tizen project using command **Tizen: Create Tizen project**..
+
+2. Open tizen_workspace.yaml file, Append character d to type:hybrid, Red line displays under hybridd and when mouse hover on it, Error message to be displayed.
+  
+   ![worksplace validation](media/rpk_tizen_workspace_yaml_open.png)
+
+   ![worksplace validation](media/rpk_tizen_workspace_yaml_validation.png)
+
+
+# Dotnet Workload Installation
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:93]: Same suggestion as line 82.
```suggestion
## Dotnet workload installation
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+1. Dotnet Workload Installation will be installed on vscode activation OR by using command **Tizen: Install Dotnet Workload**..
+
+2. For Windows OS, Run vscode as Administrator.
+
+3. For Ubuntu/Mac OS, Enter system password as it installation required sudo permission OR Press Esc to skip installation.
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:99]:
```suggestion
3. For Ubuntu/Mac OS, enter system password as the installation requires sudo permission. Or you can press Esc to skip installation.
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  
+   ![dotnet workload installation](media/dotnet_workload_install_password.png)
+
+   ![dotnet workload installation](media/dotnet_workload_install_password_enter.png)
+   
+   ![dotnet workload installation](media/dotnet_workload_install_success.png)
+   
+   > [!NOTE] 
+   > dotnet workload installion requires .NET Core SDK (.NET6 or higher).
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:108]:
```suggestion
   > Dotnet workload installation requires .NET Core SDK (.NET6 or higher).
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+   > Download and Install Latest .NET Core SDK from <https://www.microsoft.com/net/download/>
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:109]:
```suggestion
   > Download and install the latest version of .NET Core SDK from <https://www.microsoft.com/net/download/>
```

**Review by safir-srbd — COMMENTED** [review]
@rakeshkumarrajus Hello. Hope you are well.

I have reviewed the files in the PR and have provided some suggestions below. Please incorporate them at your earliest convenience.

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+   > [!NOTE] 
+   > "Resource project is Installed, But Resource project cannot be launched by Run" An info popup will be displayed.
+
+
+## Tizen workspace YAML validation
+
+1. Create Tizen project using command **Tizen: Create Tizen project**..
+
+2. Open tizen_workspace.yaml file, append character d to type:hybrid. After that a red line displays under the word 'hybridd' when the mouse is hovered on it. An error message will also be displayed.
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:86]:
```suggestion
2. Open the tizen_workspace.yaml file and append the character 'd' to type:hybrid. After that, a red line displays under the word 'hybridd' when the mouse is hovered on it. An error message will also be displayed.
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+   ![Install application](media/rpk_install_app_result.png)
+
+4. To run the application on the emulator, enter run in the **Command Palette** and select **Tizen: Run Tizen application**.
+
+   ![Run application](media/rpk_run_app1.png)
+
+   ![application](media/rpk_run_app2.png)
+
+   > [!NOTE] 
+   > "Resource project is Installed, But Resource project cannot be launched by Run" An info popup will be displayed.
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:79]:
```suggestion
   > An info popup will be displayed, showing the message "Resource project is Installed, but resource project cannot be launched by Run".
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+2. Open tizen_workspace.yaml file, append character d to type:hybrid. After that a red line displays under the word 'hybridd' when the mouse is hovered on it. An error message will also be displayed.
+  
+   ![worksplace validation](media/rpk_tizen_workspace_yaml_open.png)
+
+   ![worksplace validation](media/rpk_tizen_workspace_yaml_validation.png)
+
+
+## Dotnet workload installation
+
+1. Dotnet workload installation will be installed on vscode activation OR by using command **Tizen: Install Dotnet Workload**..
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:95]:
```suggestion
1. Dotnet workload installation will be installed on VS Code activation OR by using the command **Tizen: Install Dotnet Workload**..
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+2. For Windows OS, Run vscode as Administrator.
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:97]:
```suggestion
2. For Windows OS, Run VS Code as Administrator.
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+
+3. For Ubuntu/Mac OS, enter system password as the installation requires sudo permission. Or you can press Esc to skip installation.
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:99]:
```suggestion
3. For Ubuntu/macOS, enter system password as the installation requires sudo permission. Or you can press Esc to skip installation.
```

**Review by safir-srbd — COMMENTED** [review]
@rakeshkumarrajus Thank you for implementing the suggested changes. After going through the file one more time, I have a few more suggestions. Please incorporate them at your earliest convenience.

File: docs/application/toc_all.md [hunk 11]
```diff
@@ -1132,6 +1132,7 @@
 ### [Installing Visual Studio Code Extension for Tizen](/application/vscode-ext/Tizen/dotnet.md)
 ### Tizen
 #### [Hybrid application development](/application/vscode-ext/Tizen/hybrid.md)
+#### [Rpk Application Development](/application/vscode-ext/Tizen/rpk.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:1135]:
```suggestion
#### [RPK Application Development](/application/vscode-ext/Tizen/rpk.md)
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
+# Rpk (Resource Package) Application Development
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:1]:
```suggestion
# RPK (Resource Package) Application Development
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+
+The VS Code extension installation can be done in two different ways. Refer to [VS Code extension installation](dotnet.md) for details. 
+
+## Develop application
+
+The following sections explain how to use Visual Studio Code Extension for Tizen to develop your applications.
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:7]:
```suggestion
The following sections explain how to use Visual Studio Code Extension for Tizen to develop RPK applications.
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+### Create Tizen Rpk project
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:9]:
```suggestion
### Create Tizen RPK project
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+
+To create a Tizen Rpk project:
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:11]:
```suggestion
To create a Tizen RPK project:
```

**Review by safir-srbd — COMMENTED** [review]
@rakeshkumarrajus I also think it would be more appropriate to use RPK instead of Rpk, as in other articles on Tizen Docs site, it is fully capitalized.

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+
+   ![application](media/rpk_run_app2.png)
+
+   > [!NOTE] 
+   > An info popup will be displayed, showing the message "Resource project is Installed, but resource project cannot be launched by Run".
+
+
+## Tizen workspace YAML validation
+
+1. Create Tizen project using command **Tizen: Create Tizen project**..
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:84]:
```suggestion
1. Create Tizen project using command **Tizen: Create Tizen project**.
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+2. Open the tizen_workspace.yaml file and append the character 'd' to type:hybrid. After that, a red line displays under the word 'hybridd' when the mouse is hovered on it. An error message will also be displayed.
+  
+   ![worksplace validation](media/rpk_tizen_workspace_yaml_open.png)
+
+   ![worksplace validation](media/rpk_tizen_workspace_yaml_validation.png)
+
+
+## Dotnet workload installation
+
+1. Dotnet workload installation will be installed on VS Code activation OR by using the command **Tizen: Install Dotnet Workload**..
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:95]:
```suggestion
1. Dotnet workload installation will be installed on VS Code activation OR by using the command **Tizen: Install Dotnet Workload**.
```

File: docs/application/vscode-ext/Tizen/rpk.md [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+
+The VS Code extension installation can be done in two different ways. Refer to [VS Code extension installation](dotnet.md) for details. 
+
+## Develop application
+
+The following sections explain how to use Visual Studio Code Extension for Tizen to develop RPK applications.
+
+### Create Tizen RPK project
+
+To create a Tizen RPK project:
```
Comment (safir-srbd) [inline docs/application/vscode-ext/Tizen/rpk.md:11]:
```suggestion
To create a Tizen RPK project, follow these steps:
```

**Review by safir-srbd — COMMENTED** [review]
@rakeshkumarrajus Thank you for incorporating the suggested changes. After one last check, I found a few minor issues that need to be fixed, after which the PR can be approved. Please incorporate them at your convenience.

**Review by safir-srbd — APPROVED** [review]
@rakeshkumarrajus Thank you for implementing all the suggested changes.

@mijin85cho @asm-foysal I am approving this PR for merging.

**Comment by asm-foysal** [comment]
@safir-srbd thank you for your diligent review, I'm merging this PR.

**Pull request merged**